src/components/activity-card/activity-card.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  ActivityBody,
+  RecentEventsBody,
+} from '@console/dynamic-plugin-sdk/provisional';
+import {
+  useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import { FirehoseResource } from "@console/dynamic-plugin-sdk";
+import {getNamespace} from "../../selectors/index";
+//import { IBM_STORAGE_ODF_OPERATOR } from '../../constants/index';
+import {
+   PersistentVolumeClaimModel, 
+   EventModel,    
+  } from '../../models';
+import './activity-card.scss';
+import {GetFlashSystemResource} from '../../constants/resources';
+import { StorageInstanceKind, EventKind } from '../../types';
+
+const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+
+const RecentEvent = (props) =>{
```

Comment:
A miss here for types.
```suggestion
const RecentEvent : React.FC= (props) =>{
```